### DISCUSSION TITLE
dcutr in examples executes incorrectly

### DISCUSSION THREAD

**Discussion by xfengw** [libp2p/rust-libp2p#4385] [Q&A]
dcutr.exe --mode listen --secret-key-seed 42 --relay-address /ip4/127.0.0.1/tcp/9705
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Other(Custom { kind: Other, error: Transport(Left(Left(Left(Left(MissingDstPeerId))))) })', examples\dcutr\src\main.rs:248:18

**Comment by thomaseizinger** [comment]
The example works correctly, you are missing the `/p2p` component of the relay's address.
